fabric-object-builders-v1/src/main/java/net/fabricmc/fabric/impl/object/builder/FabricEntityType.java

```diff
@@ -25,35 +25,35 @@
 	private final int maxTrackDistance, trackTickInterval;
 	private final Boolean alwaysUpdateVelocity;
 
-	public FabricEntityType(EntityType.EntityFactory<T> factory, EntityCategory category, boolean bl, boolean summonable, boolean fireImmune, boolean spawnableFarFromPlayer, int i, int j, EntityDimensions entityDimensions, int maxTrackDistance, int trackTickInterval, Boolean alwaysUpdateVelocity) {
-		super(factory, category, bl, summonable, fireImmune, spawnableFarFromPlayer, i, j, entityDimensions);
+	public FabricEntityType(EntityType.EntityFactory<T> factory, EntityCategory category, boolean saveable, boolean summonable, boolean fireImmune, boolean spawnableFarFromPlayer, int maxDespawnDistance, int minDespawnDistance, EntityDimensions dimensions, int maxTrackDistance, int trackTickInterval, Boolean alwaysUpdateVelocity) {
+		super(factory, category, saveable, summonable, fireImmune, spawnableFarFromPlayer, maxDespawnDistance, minDespawnDistance, dimensions);
 		this.maxTrackDistance = maxTrackDistance;
 		this.trackTickInterval = trackTickInterval;
 		this.alwaysUpdateVelocity = alwaysUpdateVelocity;
 	}
 
 	@Override
 	public int getMaxTrackDistance() {
-		if (maxTrackDistance != -1) {
-			return (maxTrackDistance + 15) / 16;
+		if (this.maxTrackDistance != -1) {
```

Comment:
adding this is superfluous IMO, it is ok (=optional) for new code, but changing existing code is not necessary.

Reply:
i509VCB when he uses the Sponge formatter :stuck_out_tongue:

Reply:
I think this change was a style preference for me where I use `this` religiously when accessing fields which are a part of the class.